src/file-uploader.scss

```diff
@@ -0,0 +1,65 @@
+@import "./mixins";
+@import "./new-settings";
+@import "./button.scss";
+
+$block: #{$fd-namespace}-file-uploader;
+
+.#{$block} {
+  $fd-file-uploader-padding: 0.25rem;
+
+  @include fd-reset();
+
+  display: inline-flex;
+  align-items: center;
+  width: auto;
+
+  &__input.#{$fd-namespace}-input {
+    width: auto;
+    margin-right: $fd-file-uploader-padding;
+
+    @include fd-ellipsis();
+
+    user-select: none;
+
+    @include fd-rtl() {
+      margin-left: $fd-file-uploader-padding;
+      margin-right: 0;
+    }
+
+    cursor: text;
+
+    &:read-only {
```

Comment:
Could you use mixin called `fd-readonly` there?

Reply:
Sorry missed these Used now.